example-app/SantokuApp/src/framework/backend/refreshCsrfToken.ts

```diff
@@ -0,0 +1,13 @@
+import {BACKEND_AXIOS_INSTANCE_WITHOUT_REFRESH_SESSION, setCsrfTokenHeader} from './customInstance';
+
+type CsrfTokenResponse = {
+  csrfTokenHeaderName: string;
+  csrfTokenValue: string;
+};
+
+const refreshCsrfToken = async () => {
+  const axiosResponse = await BACKEND_AXIOS_INSTANCE_WITHOUT_REFRESH_SESSION.get<CsrfTokenResponse>('/csrf_token');
```

Comment:
```suggestion
  const axiosResponse = await BACKEND_AXIOS_INSTANCE.get<CsrfTokenResponse>('/csrf_token');
```

CSRFトークン取得は認証チェックをしないAPIなので、`BACKEND_AXIOS_INSTANCE_WITHOUT_REFRESH_SESSION` を使わなくてもいいかと思いますが、問題ありますでしょうか。

Reply:
仕様変更など何らかの理由で万が一401が返ってくることを想定しなくてよければBACKEND_AXIOS_INSTANCEでも構わないと思いますが、逆にこちらだとまずい理由はあるでしょうか。